src/Symfony/Component/Translation/Tests/DataCollectorTranslatorTest.php

```diff
@@ -34,6 +34,7 @@ public function testCollectMessages()
             'id' => 'foo',
             'translation' => 'foo (en)',
             'locale' => 'en',
+            'fallbackLocale' => '',
```

Comment:
it's `null` right? Perhaps worth to update to `assertSame` instead of `assertEquals`

Reply:
Good point,  I fixed it. But I didn't switch to assertSame, as this also checks the ordering of all items which is annoying and useless.